affordable-housing/housing.html

```diff
@@ -0,0 +1,901 @@
+<!DOCTYPE html>

```

Comment:
I created the housing.html and housing.kml files only to illustrate the sample outputs. I am not sure if you want me to move both of them to the gh-pages branch.